db/seeds.rb

```diff
@@ -12,103 +12,120 @@
+                             categories: [seafood_cat, entree_cat],
+                             image: File.open('public/FiletOfMermaid.jpg'))
 
 minotaur       = Item.create(title: "NY Strip Minotaur",
                              description: "A dish served as rare as the beast itself",
                              price: 19.99,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/NYStripMinotaur.jpg'))
 
 scylla         = Item.create(title: "Scyllamari",
                              description: "Trust us when we say this dish isn't 'Charybian'",
                              price: 17.50,
                              status: "retired",
-                             categories: [seafood_cat, entree_cat])
+                             categories: [seafood_cat, entree_cat],
+                             image: File.open('public/Scyllamari.jpg'))
 
 griffin_wings  = Item.create(title: "Buffalo Griffin Wings",
                              description: "These little wings pack an especially powerful--maybe even majestic--punch",
                              price: 6.00,
                              status: "active",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/BuffaloGriffinWings.jpg'))
 
 pegasus_wings  = Item.create(title: "Buffalo Pegasus Wings",
                              description: "One little bit will set your tastebuds aflight",
                              price: 6.00,
                              status: "retired",
-                             categories: [meat_cat, appetizer_cat])
+                             categories: [meat_cat, appetizer_cat],
+                             image: File.open('public/BuffaloPegasusWings.jpg'))
 
 leg_of_griffin = Item.create(title: "Leg of Griffin",
                              description: "Suffice it to say the king of beasts makes a meal fit for kings",
                              price: 9.50,
                              status: "active",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/LegOfGriffin.jpg'))
 
 leg_of_pegasus = Item.create(title: "Leg of Pegasus",
                              description: "This dish will send you galloping off to tasty bliss",
                              price: 9.50,
                              status: "retired",
-                             categories: [meat_cat, entree_cat])
+                             categories: [meat_cat, entree_cat],
+                             image: File.open('public/LegOfPegasus.jpg'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.